Tools/Google.Apis.Release/Resources/Google.Apis.Auth.VERSION.nuspec

```diff
@@ -25,7 +25,8 @@
         </description>
         <tags>Google</tags>
         <dependencies>
-            <dependency id="Google.Apis.Core" version="VERSION" />
+          <dependency id="BouncyCastle" version="1.7.0" />

```

Comment:
Please keep the spacing as it was before (I think you need to add two spaces in line 28 and 29).